src/sidebar/components/annotation-omega.js

```diff
@@ -78,6 +79,15 @@ function AnnotationOmega({
     createDraft(annotation, { ...draft, text });
   };
 
+  const onReply = () => {
+    // TODO: Re-evaluate error handling here
+    if (!userid) {
+      flash.error('Please log in to reply to annotations');
+    } else {
+      annotationsService.reply(annotation, userid);
```

Comment:
Since the `userid` is just a value that comes from the store, which the annotations service already uses, perhaps the annotations service could look this up itself rather than passing it to the `reply` method? This component would need to use `store.isLoggedIn()` to check that the user is logged in that case.

Reply:
Perhaps I could make it an optional arg and have the service look it up if it's not present? That leaves the ability for a caller to supply a user other than the currently-logged-in user to be the author of the reply. No, I do not in fact have an example of how this would be useful :D. Perhaps I'm overgeneralizing.